ui/ui-components/pages/model/[modelId]/source/[sourceId]/schedule.tsx

```diff
@@ -58,6 +58,11 @@ export default function Page(props) {
       _schedule.recurringFrequency = recurringFrequencyMinutes * (60 * 1000);
     }
 
+    let thing = Object.assign({}, _schedule, {
+      filters: localFilters,
+      state: "ready",
+    });
```

Comment:
what's *the thing*?!

Reply:
You know... The _thing_.

(A line I used for debugging that I forgot to delete)